common/app/model/Tag.scala

```diff
@@ -286,4 +287,8 @@ case class Tag (
   val isFootballTeam = properties.references.exists(_.`type` == "pa-football-team")
   val isFootballCompetition = properties.references.exists(_.`type` == "pa-football-competition")
   val contributorImagePath = properties.bylineImageUrl.map(ImgSrc(_, Contributor))
+
+  override def branding(edition: Edition): Option[Branding] = {
```

Comment:
`Tag` and `Section` are both types of `Page`, and they both have a `branding` method which works out the branding based on the edition in scope.